src/tests/Common/CoreCLRTestLibrary/PlatformDetection.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+public static class PlatformDetection
```

Comment:
Can we put this in the `TestLibrary` namespace instead of the global namespace? I know you'd have to go back and manually update all of the custom attributes, so I understand if you'd rather not do so.

Reply:
No worries, I can easily do that.